app/Database/Requirement.hs

```diff
@@ -12,28 +12,28 @@ module Requirement
 ) where
 
 import qualified Data.String as S
+import qualified Data.List as L
 
-
-data ProgramReq = PRGREQ String [Req]
```

Comment:
I'm not sure why you changed this; it seems to be the case that a program will have multiple requirements. I guess you could represent this as an implicit "AND", but it will be useful to keep them separate at the top level, I think.